ChatQnA/README.md

```diff
@@ -18,17 +18,81 @@ This ChatQnA use case performs RAG using LangChain, Redis VectorDB and Text Gene
 
 The ChatQnA service can be effortlessly deployed on either Intel Gaudi2 or Intel XEON Scalable Processors.
 
+Currently we support two ways of deploying ChatQnA services with docker compose:
+
+1. Start services using the docker image on `docker hub`:
+
+```bash
+docker pull opea/chatqna:latest
+```
+
+Two type of UI are supported now, choose one you like and pull the referred docker image.
+
+If you choose conversational UI, follow the [instruction](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA/docker/gaudi#-launch-the-conversational-ui-optional) and modify the [docker_compose.yaml](./docker/xeon/docker_compose.yaml).
```

Comment:
docker compose will start both ui images, https://github.com/opea-project/GenAIExamples/blob/8f00ae6a6c442ffb728a6fc311ae7720292fa8f2/ChatQnA/docker/xeon/docker_compose.yaml#L192

Reply:
We will raise another PR to keep only one ui image.